### PR TITLE
[RM Ch06] Corrections from GSMA review

### PR DESCRIPTION
**Pull request by walterkozlowski**

### CODE REVIEW

**Review by CsatariGergely — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Comment by collivier** [comment]
@walterkozlowski do u need help to fix this issues?

**Comment by walterkozlowski** [comment]
@collivier, thanks for the help offer.  Yes, please.  Very much appreciated.

File: doc/ref_model/chapters/chapter06.rst [hunk 1]
```diff
@@ -210,15 +210,15 @@ These acceleration interfaces are summarized here in Table 6.3 only for convenie
 |                       |                        |       |        | ata           |                                    |
 +-----------------------+------------------------+-------+--------+---------------+------------------------------------+
 | DeRegisterForAccEvent | DeRegisterForAccEventR | VNF → | Input  | accEvent      | Event VNF is deregistering from    |
-| Response              | esponse                | NFVI  |        |               |                                    |
+| Request               | esponse                | NFVI  |        |               |                                    |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter06.rst:213]:
```suggestion
| Request               | Response                | NFVI  |        |               |                                    |
```

Comment (collivier) [reply doc/ref_model/chapters/chapter06.rst:213]: Pankaj it's incorrectly wrapped (see line above).

is it DeRegisterForAccEvent Response or DeRegisterForAccEventResponse ?
I think many slashes are missing in this tabs.

I think they must be fixed in another PR.

Comment (collivier) [reply doc/ref_model/chapters/chapter06.rst:213]: @pgoyal01 @walterkozlowski https://github.com/cntt-n/CNTT/issues/3177

**Comment by collivier** [comment]
+ https://github.com/cntt-n/CNTT/issues/3177

**Review by collivier — APPROVED** [review]
https://github.com/cntt-n/CNTT/issues/3177

**Pull request merged**